biojava-structure/src/main/java/org/biojava/nbio/structure/secstruc/BasePairParameters.java

```diff
@@ -0,0 +1,403 @@
+package org.biojava.nbio.structure.secstruc;
```

Comment:
Ok, I am going to call the package basepairs, since it's obtaining both the pairing parameters (buckle, propeller, opening, shear, stretch, and stagger), and the base-pair step parameters (tilt, roll, twist, shift, slide, and rise).  Today I will work on JavaDoc for everything that the user might want to customize or override.